static/js/app.js

```diff
@@ -1203,6 +1203,32 @@ function bindTableHeaderMenu() {
       var menuItem = $(e.target);
 
       switch(menuItem.data("action")) {
+        case "copy_as_insert":
```

Comment:
Can you wrap functionality from `copy_as_*` into a separate function to keep the handler function small?